### PR TITLE
[hotfix][docs] Fix all broken images

### PR DESCRIPTION
**Pull request by Shawn-Hx**
## What is the purpose of the change

Fix all broken images after migrating to Hugo.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by Shawn-Hx** [comment]
Hi, @xintongsong, @sjwiesman 
I find several broken images in docs and fix them. PTAL :)

**Comment by xintongsong** [comment]
Thanks @Shawn-Hx, I'll take a look.

A quick question, how did you locate these broken images? Did you just manually go through all the pages, or is there an automatic approach? I'm asking because, if there's an automatic approach, we could introduce a ci-check against broken images.

**Comment by Shawn-Hx** [comment]
Thanks @xintongsong .

Before migrating Flink docs from Jekyll to Hugo, I have changed the link format in all markdown files in FLINK-20298.
The link format in Jekyll likes "{% link ... %}", which is different from Hugo. So I use grep command to locate all matched text in the docs directory. This is a simple way to find all broken images.
Now I'm not sure if Hugo have some tools that can find all broken links automatically.

File: docs/content.zh/docs/try-flink/table_api.md
Comment (xintongsong) [inline docs/content.zh/docs/try-flink/table_api.md:307]: Maybe replace this with `{{}< img xxx>}` and preserve the height & width.
```suggestion
{{< img src="/fig/spend-report-console.png" height="400px" width="800px" alt="Flink Console">}}
```

File: docs/content/docs/try-flink/table_api.md
Comment (xintongsong) [inline docs/content/docs/try-flink/table_api.md:307]: Maybe replace this with `{{}< img xxx>}` and preserve the height & width.
```suggestion
{{< img src="/fig/spend-report-console.png" height="400px" width="800px" alt="Flink Console">}}
```

**Comment by xintongsong** [comment]
Thanks for the information, @Shawn-Hx and @sjwiesman.
Sound like both of you were checking against legacy Jekyll formats. I guess it's fine that we can't automate this process, since the migration is done and ideally we should not introduce any new broken links/images.

**Review by xintongsong — APPROVED** [review]
@Shawn-Hx
The changes LGTM. I have only one minor comment. Let's wait for @sjwiesman's feedback before merging this.

**Comment by Shawn-Hx** [comment]
Hi @sjwiesman  
I have appended commits according to the review comments. PTAL.

**Comment by sjwiesman** [comment]
LGTM, Merging into master and release-1.13

**Pull request closed without merging**
